.travis.yml

```diff
@@ -21,10 +21,11 @@ cache:
 before_script:
   # default $SBT_OPTS is irrelevant to sbt lancher
   - unset SBT_OPTS
+  # ensure the build errors if dependencies aren't available
+  - travis_retry ./bin/sbt ++$TRAVIS_SCALA_VERSION update
```

Comment:
keyword retry! so whenever maven.twttr.com fails (and yes.. it does often).. we might actually be able to proceed.